lib/prefix.h

```diff
@@ -499,11 +499,8 @@ extern int macstr2prefix_evpn(const char *str, struct prefix_evpn *p);
 /* NOTE: This routine expects the address argument in network byte order. */
 static inline bool ipv4_martian(const struct in_addr *addr)
 {
-	in_addr_t ip = ntohl(addr->s_addr);
```

Comment:
zebra uses ipv4_martian.  Is this appropriate of a change?

Reply:
This is already done inside `ipv4_unicast_valid()`.